regression/cbmc-concurrency/Makefile

```diff
@@ -1,10 +1,19 @@
 default: tests.log
 
+include ../../src/config.inc
+include ../../src/common
+
+ifeq ($(filter-out OSX MSVC,$(BUILD_ENV_)),)
+	# no POSIX threads on Windows
+	# for OSX we'd need sound handling of pointers in multi-threaded programs
```

Comment:
? Why is that concern particular to OSX?

Reply:
I believe the difference is whether `pthreadt` is a typedef for a pointer type or not, which varies from platform to platform.